diff --git a/node_modules/vue-template-compiler/index.js b/node_modules/vue-template-compiler/index.js
index dcc7e6c..3970c19 100644
--- a/node_modules/vue-template-compiler/index.js
+++ b/node_modules/vue-template-compiler/index.js
@@ -7,14 +7,14 @@ var packageVersion = require('./package.json').version
 if (vueVersion && vueVersion !== packageVersion) {
   var vuePath = require.resolve('vue')
   var packagePath = require.resolve('./package.json')
-  throw new Error(
-    '\n\nVue packages version mismatch:\n\n' +
-    '- vue@' + vueVersion + ' (' + vuePath + ')\n' +
-    '- ' + packageName + '@' + packageVersion + ' (' + packagePath + ')\n\n' +
-    'This may cause things to work incorrectly. Make sure to use the same version for both.\n' +
-    'If you are using vue-loader@>=10.0, simply update vue-template-compiler.\n' +
-    'If you are using vue-loader@<10.0 or vueify, re-installing vue-loader/vueify should bump ' + packageName + ' to the latest.\n'
-  )
+  // throw new Error(
+  //   '\n\nVue packages version mismatch:\n\n' +
+  //   '- vue@' + vueVersion + ' (' + vuePath + ')\n' +
+  //   '- ' + packageName + '@' + packageVersion + ' (' + packagePath + ')\n\n' +
+  //   'This may cause things to work incorrectly. Make sure to use the same version for both.\n' +
+  //   'If you are using vue-loader@>=10.0, simply update vue-template-compiler.\n' +
+  //   'If you are using vue-loader@<10.0 or vueify, re-installing vue-loader/vueify should bump ' + packageName + ' to the latest.\n'
+  // )
 }
 
 module.exports = require('./build')
